api/src/models/user.model.js

```diff
@@ -5,8 +5,8 @@ const mongoose = require('mongoose');
 
 
 const userSchema = new mongoose.Schema({
-    givenName: { type: String, required: false },
-    familyName: { type: String, required: false },
+    teamName: { type: String, required: true },
+    orgName: { type: String, required: false },
```

Comment:
What's the difference between `teamName` and `orgName`? We should document this somewhere (e..g https://docs.google.com/document/d/1U68bJV0POrh_wDXem_hZ_PrivoVT6qZCPC8vw7Dg5uQ/edit)

Reply:
`teamName` - name of the research group (likely chosen by the members)
`orgName` - the Organisation name the group is part of, like Monash university, CSL, etc.
I have updated this in the document you linked.